client/js/components/map/publicdetail/Map.vue

```diff
@@ -420,7 +420,12 @@ export default {
         const firstTogglableLayer = this.baseLayers.find(layer => {
           return layer.getProperties().isEnabled === true && layer.getProperties().title !== 'customBaselayerDanmark'
         })
-        firstTogglableLayer.setVisible(true)
+
+        if(typeof firstTogglableLayer !== 'undefined') {
```

Comment:
This will be executed if there are baseLayers defined, but none of them is published. Just out of curiosity: did you test how the system behaves if no baseLayer is defined at all?

Apart from that, a minor nitpicky suggestion:

```suggestion
        if (typeof firstTogglableLayer !== 'undefined') {
```

Reply:
done https://github.com/demos-europe/demosplan-core/pull/626/commits/f105c540478e6563fec85e81b788b9d3382c1754